### DISCUSSION TITLE
Fail to start databend-query process for debugging

### DISCUSSION THREAD

**Discussion by Jake-00** [databendlabs/databend#11536] [Q&A]
## Description
Following [building instruction](https://databend.rs/doc/contributing/building-from-source), everything goes well before section **[Start Databend for Debugging](https://databend.rs/doc/contributing/building-from-source#start-databend-for-debugging)**. After typing the command below in terminal, databend-meta process can run healthly, but databend-query would exit with code  `209`. 
```
# 1. Start databend-meta first:
nohup target/debug/databend-meta --single --log-level=ERROR & 

# 2. Start databend-query and connect it to databend-meta:
nohup target/debug/databend-query -c scripts/ci/deploy/config/databend-query-node-1.toml &
```
Moreover, two sentences appear at the end of **nohup.out** file
```
Databend Query start failure, cause: Code: 2001, Text = meta-client dedicated runtime error: tokio::sync::oneshot::error::RecvError: channel closed while: when recv resp from MetaGrpcClient worker.

<Backtrace disabled by default. Please use RUST_BACKTRACE=1 to enable>
```

## Expectation
As firstly working with this project, I need some help to figure out this problem.

**Comment by wubx** [comment]
Get the latest tags before compiling
```
git fetch https://github.com/datafuselabs/databend.git --tags
```
